lib/tasks/sync_github.rake

```diff
@@ -0,0 +1,28 @@
+namespace :sync_github do
+
+  # This task fetch issues details and create author and book on our DB 
+  desc "Sync db from github"
+  task :author_details_from_github => :environment do
+    issues = Github.new.issues.list user: ENV['USERNAME'], repo: ENV['REPOSITORY']
+    issues.each do |issue|
+      author = Author.find_by_github_id(issue["number"])    
+      if author.blank?
+        author = Author.create(name: issue["title"], biography: issue["body"], github_id: issue["number"])
+        author.books.create(title: Faker::Book.title, publisher: author)
```

Comment:
This code is duplicated from your controller, how would you go about avoiding the repetition? I know this is just two lines right now, but in a real world example it's very likely that the business logic around creating a new author would expand over time, and this duplication increases the likelihood of the two paths diverging if devs forget to update one place or the other.

Reply:
Move this code to `Author` model and noted for the next time
Thanks!